### PR TITLE
News application - Pinned news

### PR DESCRIPTION
**Pull request by samehaidi**

### CODE REVIEW

File: docs/GettingStarted.rst [hunk 1]
```diff
@@ -1361,6 +1363,18 @@ You can look for articles easily using the available search bar in the topbar of
 
 |image191|
 
+
+News application views
```
Comment (thomasdelhomenie) [inline docs/GettingStarted.rst]: I would use `filters` instead of `views`. WDYT ?

File: docs/GettingStarted.rst [hunk 2]
```diff
@@ -1361,6 +1363,18 @@ You can look for articles easily using the available search bar in the topbar of
...
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Different views are available from News application so you can easily find pinned articles, your posted articles, your scheduled articles, your drafts or your archived articles.
```
Comment (thomasdelhomenie) [inline docs/GettingStarted.rst]: You anticipate, right ? :)

File: docs/GettingStarted.rst [hunk 3]
```diff
@@ -1361,6 +1363,18 @@ You can look for articles easily using the available search bar in the topbar of
...
+
+To display the pinned news list, all you have to do is to choose this option from the dropdown menu of the application topbar.
```
Comment (thomasdelhomenie) [inline docs/GettingStarted.rst]:
```suggestion
To display the pinned news list, select this option from the dropdown menu of the application topbar.
```

**Review by thomasdelhomenie — CHANGES_REQUESTED** [review]

File: docs/GettingStarted.rst (hunk 1 above)
Comment (samehaidi) [reply docs/GettingStarted.rst]: I wrote Views to distinguish between filtering by spaces and the different views availables from the dropdown menu as it is not the same used component.

File: docs/GettingStarted.rst (hunk 2 above)
Comment (samehaidi) [reply docs/GettingStarted.rst]: Yes. To have the intro for the next documentation PRs.

File: docs/GettingStarted.rst (hunk 1 above)
Comment (thomasdelhomenie) [reply docs/GettingStarted.rst]: Yes, but functionally speaking it is still filters. Talking about views make me think of different layouts for the News list.

Comment (samehaidi) [reply docs/GettingStarted.rst]: OK. I updated the text

**Review by thomasdelhomenie — APPROVED** [review]

**Pull request merged**